server.js

```diff
@@ -7,9 +7,32 @@ const express = require("express");
 const mongoose = require("mongoose");
 
 
-mongoose.connect(`mongodb+srv://${process.env.NAME}:${process.env.PASSWORD}@cluster0.jfqke.mongodb.net/myFirstDatabase?retryWrites=true&w=majority`
+if (process.env.NODE_ENV === "production") {
+
+    // ---- For running in production: -----
+    mongoose.connect(`mongodb+srv://${process.env.NAME}:${process.env.PASSWORD}@cluster0.jfqke.mongodb.net/myFirstDatabase?retryWrites=true&w=majority`
     , { useNewUrlParser: true, useUnifiedTopology: true, useCreateIndex: true });
 
+} else if (process.env.NODE_ENV === 'development') {
+
+    /* ---- For local testing: -----
+    1. run the comands:
```

Comment:
Instead of having the instructions here lets just put this in the readme. Other than that looks good to go.